R/distances.R

```diff
@@ -419,41 +430,104 @@ distancesMatrix <- function(t.layer, starters = NULL, targets = starters,
     length(suppressWarnings(packageDescription("terra"))))
   missing.packages <- sapply(aux, function(x) x == 1)
   if (any(missing.packages)) {
-    stop(paste0("This function requires packages '", paste(c("raster", "gdistance", "sp", "terra")[missing.packages], collapse = "', '"),
-      "' to operate. Please install ", ifelse(sum(missing.packages) > 1, "them", "it"), " before proceeding.\n"), call. = FALSE)
+    stop(paste0("This function requires packages '", 
+                paste(c("raster", "gdistance", "sp", "terra")[missing.packages], 
+                      collapse = "', '"),
+      "' to operate. Please install ", 
+      ifelse(sum(missing.packages) > 1, "them", "it"), 
+      " before proceeding.\n"), call. = FALSE)
```

Comment:
```suggestion
    stop("This function requires packages '", 
         paste(c("raster", "gdistance", "sp", "terra")[missing.packages], 
               collapse = "', '"),
         "' to operate. Please install ", 
         ifelse(sum(missing.packages) > 1, "them", "it"), 
         " before proceeding.\n", call. = FALSE)
```

Taking the chance to remove an unnecessary paste0()